bin/fetch_publishing_api_links.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+
+# Extracts content items with their links from the publishing API database
+# (about 200,000 items) and dumps the data to disk as CSV.
+
+require 'pg'
+require 'csv'
+
+if ARGV.length < 1
+  $stderr.puts "Usage: #{$0} output_file"
+  exit 1
+end
+
+connection = PG.connect(ENV["DATABASE_URL"] || 'postgresql:///publishing_api_development')
+
+query = <<-QUERY
+SELECT
+  c.content_id,
```

Comment:
Could we write more meaningful names than `c`, `lo` ...
I feel it's too many of them to follow without confusing myself, specially `l`, `ls`, `lo`

What about replacing them with

```
c -- > item
l  --> link
lo --> location
ls --> link_set
s  --> state
etc..
```